edxadapt/static/js/src/edxadapt.js

```diff
@@ -65,7 +65,8 @@ function EdxAdaptXBlock(runtime, element) {
         })
         .fail(function(jqXHR, textStatus) {
             if (jqXHR.status == 404) {
-                // Student not found in EdxAdapt. Let's register them
+                // Student not found in EdxAdapt. Let's register him
+                configureUser();
                 registerUser();
```

Comment:
These are async functions so to ensure execution order you may want to move `registerUser()` call inside `.done` section of `configureUser()`.